### PR TITLE
Handle escape menu with toggle iteractMenu

### LINKED ISSUE #5136: Toggle interaction menu and esc button issues
**Arma 3 Version:** `1.68` (stable)
**CBA Version:** `3.3.1` (stable)
**ACE3 Version:** `3.9.1` (dev + c001be5b09d749737551acf281de70d5c8c32073)

**Mods:**
```
- CBA_A3
- ace
```

**Description:**
- With cursor enabled: pressing esc hides the menu (if show cursor is enabled) but leaves the black-screen background (if enabled, of course). Without cursor enabled: esc just opens esc-menu. Would be nice if esc just closed the action menu in both cases.

**Steps to reproduce:**
- Set to toggle. Open menu. Press esc.

**Where did the issue occur?**
- Dedicated / Self-Hosted Multiplayer / Singleplayer / Editor (Singleplayer) / Editor (Multiplayer) / Virtual Arsenal

**Placed Modules:**
- None.

**RPT log file:**
- N/A.

### PR DESCRIPTION
**Pull request by PabstMirror**
Close #5136

### CODE REVIEW

**Comment by rebelvg** [comment]
Just tried this. Interaction with cursor on closes properly now, but interaction without curson still opens esc menu.

**Comment by PabstMirror** [comment]
This should work on both display options now.


There is a tiny-micro stutter when hitting escape because it partialy loads the escape menu before closing, not ideal but it's barley noticeable.

**Comment by rebelvg** [comment]
Yep, seems to work.
Strange behavior can be noticed when you open interaction with cursor and while it's open you try to open interaction without cursor. It will kinda close previous interaction but won't open other interaction and still have black overlay enabled with the cursor.

File: addons/interact_menu/functions/fnc_handleEscapeMenu.sqf [hunk 1]
```diff
@@ -0,0 +1,25 @@
+/*
+ * Author: PabstMirror
+ * Handle the escape key being pressed
```
Comment (jonpas) [inline addons/interact_menu/functions/fnc_handleEscapeMenu.sqf]: Missing final dot.

**Pull request merged**